JitsiConference.js

```diff
@@ -47,6 +46,11 @@ function JitsiConference(options) {
     this._init(options);
     this.componentsVersions = new ComponentsVersions(this);
     this.participants = {};
+    /**
+     * Jingle session instance for the JVB connection.
+     * @type {JingleSessionPC}
+     */
+    this.jvbJingleSession = null;
```

Comment:
It was in the ChatRoom. I moved it in this commit: bee0b28
Brian asked the same question and you should be able to find our discussion above (sorry I don't know how to paste link to it)